tests/unit/test_api_delete.py

```diff
@@ -0,0 +1,12 @@
+"""Unit tests for DELETE /ctms/{primar_email}"""
+
+
+def test_delete_contact_by_primary_email_not_found(client):
+    resp = client.delete("/ctms/foo@bar.com")
+    assert resp.status_code == 404
+
```

Comment:
Can you add a test that verifies that an anonymous client cannot delete also please?

Reply:
yes, scratch that, sorry!